### PR TITLE
Validation added on Edit profile section

### PR DESCRIPTION
**Pull request by Geeta259**
issue #1147 
Empty validation check for each field.
Error message highlighted below of each field.

Screenshot (67)
Screenshot (68)
Screenshot (71)
Screenshot (73)
Screenshot (74)

@narayan954 Please look out this PR linked with issue #1147

### CODE REVIEW

File: src/components/EditProfile/index.jsx [hunk 1]
```diff
@@ -78,38 +78,104 @@ const EditProfile = ({ userData, username, setIsEditing, setUserData }) => {
     }
   };
 
-  const handleImgSave = () => {
-    if (!usernameAvailable) {
-      return;
+  const [error, setError] = useState({});
```
Comment (narayan954) [inline src/components/EditProfile/index.jsx:81]: define usestate in the top along with others to maintain consistency

File: src/components/EditProfile/index.jsx [hunk 2]
```diff
@@ -78,38 +78,104 @@ const EditProfile = ({ userData, username, setIsEditing, setUserData }) => {
...
+  const handleImgSave = (event) => {
+    event.preventDefault();
+    console.log("called");
+    console.log(editedData.name);
```
Comment (narayan954) [inline src/components/EditProfile/index.jsx:85]: remove console logs as they're not required in production.... (though maybe I should add a plugin later to automatically remove them)

**Review by narayan954 — CHANGES_REQUESTED** [review]
some minor changes

File: src/components/EditProfile/index.jsx [hunk 3]
```diff
@@ -78,65 +78,124 @@ const EditProfile = ({ userData, username, setIsEditing, setUserData }) => {
...
+    if (editedData.name === "") {
+      setError("**Name is Required!");
+    } else if (editedData.newUsername === "") {
+      setError("**User name is Required!");
+    } else if (editedData.country === "") {
+      setError("**Country is Required!");
+    } else if (editedData.bio === "") {
+      setError("**Bio is Required!");
+    } else {
```
Comment (narayan954) [inline src/components/EditProfile/index.jsx:94]: also, can we pick the above logic in a separate function to improve readibility?

**Comment by narayan954** [comment]
> issue #1147 Empty validation check for each field. Error message highlighted below of each field.
> 
> Screenshot (67) Screenshot (68) Screenshot (71) Screenshot (73) Screenshot (74)
> 
> @narayan954 Please look out this PR linked with issue #1147

also, perhaps country and bio could be optional right?

**Comment by narayan954** [comment]
is it ready? Can you please resolve conflicts?

**Comment by Geeta259** [comment]
> @narayan954 Please look out this PR linked with issue #1147

check out this PR #1205

**Comment by narayan954** [comment]
> > @narayan954 Please look out this PR linked with issue #1147
> 
> check out this PR #1205

alright, closing this duplicate pr

**Pull request closed without merging**